api/core/tools/provider/builtin/xtuis/xtuis.yaml

```diff
@@ -0,0 +1,29 @@
+identity:
+  author: jimmy wu
+  name: xtuis
+  label:
+    en_US: xtuis
+    zh_Hans: 虾推啥
+    pt_BR: xtuis
+  description:
+    en_US: send message to Weixin/Wechat
+    zh_Hans: 一行代码发送信息到微信
+    pt_BR: send message to Weixin/Wechat
+  icon: icon6.png
```

Comment:
This should be `.svg`. 
Please rename `icon6.svg` to `xtuis.svg`.

Reply:
> This should be `.svg`. Please rename `icon6.svg` to `xtuis.svg`.

thanks. i have rename to xtuis.svg, and also update the xtuis.yaml with new icon names. pls kindly review it again.